mixer/cmd/bundles.go

```diff
@@ -114,10 +114,55 @@ var bundleListCmd = &cobra.Command{
 	},
 }
 
+// Bundle Edit command ('mixer bundle edit')
+type bundleEditCmdFlags struct {
+	noEdit bool
+	editor string
+	add    bool
+}
+
+var bundleEditFlags bundleEditCmdFlags
+
+var bundleEditCmd = &cobra.Command{
+	Use:   "edit [bundle(s)]",
+	Short: "Edit local and upstream bundles",
+	Long: `Edit local and upstream bundle definition files. This command will locate the
+bundle (looking first in local-bundles, then in upstream-bundles), and launch
+an editor to edit it. If the bundle is only found upstream, the bundle file will
+first be copied to your local-bundles directory for editing. When the editor
+closes, the bundle file is then parsed for validity.
+
+The editor is configured in builder.conf via the BUNDLE_EDITOR field, or by flag
+override. Valid options include: nano, vim, emacs, joe, and ctags. The default
+value is nano.
+
+Passing '--no-edit' will suppress launching the editor, and will thus only copy
```

Comment:
`mixer bundle edit --no-edit` sounds strange, maybe `mixer bundle edit --copy-only`?

Reply:
Hahah, I had that same thought last night. I couldn't come up with a better name (it was late and I was hungry), but that works pretty well.